eden/graph.py

```diff
@@ -582,7 +582,8 @@ def _weight_preprocessing(self, graph):
                 if self.key_weight not in d:
                     graph.node[n][self.key_weight] = 1
 
-    def _edge_to_vertex_transform(self, original_graph):
+    @staticmethod
+    def _edge_to_vertex_transform(original_graph):
```

Comment:
It should be static so I don't have to create a Vectorizer object each time I want to edge_to_vertex_transform or back transform.  Probably these functions should be moved out of the class entirely.   In the near future more than just these 3 functions have this problem since I want to use the graph preprocessing stuff without the Vectorizer.  Should I copy the functions I need to GraphLearn?